SLHCUpgradeSimulations/Configuration/python/phase2TkFlat.py

```diff
@@ -27,6 +29,48 @@ def customise(process):
 
     return process
 
+
+def customise_DigiTkOnly(process):
+    process.load('Configuration.StandardSequences.Digi_cff')
+    process.doAllDigi = cms.Sequence()
+    process.load('SimGeneral.MixingModule.mixObjects_cfi')
+    process.digitisationTkOnly_step.remove(process.mix.mixObjects.mixCH)
+    del process.simCastorDigis
+    del process.simEcalUnsuppressedDigis
+    del process.simHcalUnsuppressedDigis
+    process.mix.digitizers = cms.PSet(process.theDigitizersValid)
+    process.digitisationTkOnly_step.remove(process.mix.digitizers.ecal)
+    process.digitisationTkOnly_step.remove(process.mix.digitizers.hcal)
+    process.digitisationTkOnly_step.remove(process.mix.digitizers.castor)
```

Comment:
Erica's earlier question raised my interest. What are these lines attempting to do? `digitisationTkOnly_step` seems to be a `cms.Path` and I'd guess e.g. `process.mix.digitizers.ecal` is not an EDModule, so AFAICT the lines don't do anything.